seqr/views/apis/individual_api.py

```diff
@@ -364,6 +364,17 @@ def save_individuals_table_handler(request, project_guid, upload_file_id):
     return create_json_response(updated_families_and_individuals_by_guid)
 
 
+def add_individuals_and_families(project, individual_records, user):
```

Comment:
parsing to TSV data doesn't belong in this file, it belongs in the same file that creates the anvil project which is the only place we would ever need that functionality. In general, public helper functions should be things that are/ could easily be needed in multiple different files. Very specific functions should be private helper methods in the file where they are used.

`_add_or_update_individuals_and_families` was a function that was only ever used in this one file so it was a private helper method in the file. Now that we need it in multiple places, you should move  it to the individual_utils file and make it public (i.e. remove the leading `_`), and then import it both in this file where it is already used, and also in the anvil workspace api